framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -285,104 +283,120 @@ protected boolean areEqualInHierarchy(
   }
 
   /**
-   * A declared type is considered a supertype of another declared type only if all of the type
-   * arguments of the declared type "contain" the corresponding type arguments of the subtype.
-   * Containment is described in <a
+   * Returns true if {@code outside} contains {@code inside}, that is, if the set of types denoted
+   * by {@code outside} is a superset of or equal to the set of types denoted by {@code inside}.
+   *
+   * <p>Containment is described in <a
    * href="https://docs.oracle.com/javase/specs/jls/se11/html/jls-4.html#jls-4.5.1">JLS section
    * 4.5.1 "Type Arguments of Parameterized Types"</a>.
    *
-   * @param inside a type argument of the "subtype"
-   * @param outside a type argument of the "supertype"
+   * <p>As described in <a
+   * href=https://docs.oracle.com/javase/specs/jls/se11/html/jls-4.html#jls-4.10.2>JLS section
+   * 4.10.2 Subtyping among Class and Interface Types</a>, a declared type is considered a supertype
+   * of another declared type only if all of the type arguments of the declared type "contain" the
+   * corresponding type arguments of the subtype.
+   *
+   * @param inside a possibly-contained type
+   * @param outside a possibly-containing type
```

Comment:
What is the relationship between these types?  Are their erased/raw types the same?  Or is there a subtyping relationship?